back/engines/free/email_campaigns/app/models/email_campaigns/campaigns/invite_reminder.rb

```diff
@@ -46,7 +46,11 @@ def activity_triggers
     end
 
     def filter_recipient(users_scope, activity:, time: nil)
-      users_scope.where(id: activity.item.invitee.id)
+      if activity.item&.invitee_id
+        users_scope.where.not(email: nil).and(users_scope.where(id: activity.item.invitee_id))
```

Comment:
Did you consider adding sth like that to `back/engines/free/email_campaigns/app/models/email_campaigns/campaign.rb`?
```ruby
def self.recipient_filter(action_symbol)
  @recipient_filters ||= []
  @recipient_filters << action_symbol
end

recipient_filter :filter_out_recipient_without_email

def filter_out_recipient_without_email(users_scope, activity:, time: nil)
  users_scope.where.not(email: nil)
end
```
or some other way of filtering out users w/o an email for all campaigns in one place?